From 94993057898e1ff91bb39808e152590cb1aeaf3c Mon Sep 17 00:00:00 2001
From: Anurag Das <anurdas@codeaurora.org>
Date: Fri, 30 Nov 2018 18:33:01 +0530
Subject: [PATCH 1/2] wifi: Use stub for add_or_remove_virtual_intf
 functionality

Use the stub to take care of cases where the vendor HAL
implementation is not present. This will return an error
as NOT_SUPPORTED when the functionality is needed by framework
(with no implementation wifi hal). This will avoid the failure
scenario when vendor implementation for add_or_remove_virtual_intf
is not present in wifi hal.

Change-Id: I5c1c99acb40e721e22afa2683b5b0836e533f34e
CRs-Fixed: 2363891
---
 wifi/1.2/default/wifi_legacy_hal_stubs.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/wifi/1.2/default/wifi_legacy_hal_stubs.cpp b/wifi/1.2/default/wifi_legacy_hal_stubs.cpp
index fc28bb55..3cb9fd1a 100644
--- a/wifi/1.2/default/wifi_legacy_hal_stubs.cpp
+++ b/wifi/1.2/default/wifi_legacy_hal_stubs.cpp
@@ -137,6 +137,7 @@ bool initHalFuncTableWithStubs(wifi_hal_fn* hal_fn) {
     populateStubFor(&hal_fn->wifi_select_tx_power_scenario);
     populateStubFor(&hal_fn->wifi_reset_tx_power_scenario);
     populateStubFor(&hal_fn->wifi_set_radio_mode_change_handler);
+    populateStubFor(&hal_fn->wifi_add_or_remove_virtual_intf);
     return true;
 }
 }  // namespace legacy_hal
-- 
2.17.1

